gensrc/thrift/AgentService.thrift

```diff
@@ -122,6 +122,22 @@ struct TCreateTabletReq {
     20: optional bool create_schema_file = true;
 }
 
+struct TCreateTableReq {
```

Comment:
```suggestion
struct TTxnCreateTableReq {
```
all new req types based on new transactional req framework should start with `TTxn`